driver/task.go

```diff
@@ -8,9 +8,20 @@ import (
 	mocks "github.com/hashicorp/consul-terraform-sync/mocks/client"
 )
 
+const (
```

Comment:
As this grows it might be worthwhile to convert it to follow Go's enum-like pattern using iota with a custom type to allow for more compile time checks with these options.

Reply:
This in "this grows" meaning the list of possible options.

Reply:
Note that next time I'll put this below the constant values so they are in the displayed snippet.

Reply:
Interesting! I had seen the iota pattern around and didn't realize it was to solve the go enum problem. I'll keep this in mind. In my next PR, I'll use these values so that they can be passed as query parameter values in the API e.g. ?run=now and ?run=inspect. I'm not sure if the iota types will translate ok for that?